subsys/ipc/rpmsg_multi_instance/Kconfig

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2020-2021 Nordic Semiconductor (ASA)
+# SPDX-License-Identifier: Apache-2.0
+
+# Workaround for not being able to have commas in macro arguments
+DT_CHOSEN_Z_IPC_SHM := zephyr,ipc_shm
+DT_CHOSEN_Z_IPC := zephyr,ipc
+DT_CHOSEN_Z_IPC_TX := zephyr,ipc_tx
+DT_CHOSEN_Z_IPC_RX := zephyr,ipc_rx
```

Comment:
It seems these values are not used in this implementation
```suggestion
```

Reply:
You're right. It's not used. IPM channel names are manually filled in.